pt-BR/action_mailer_basics.md

```diff
@@ -747,13 +746,14 @@ files (environment.rb, production.rb, etc...)
 |`deliveries`|Keeps an array of all the emails sent out through the Action Mailer with delivery_method :test. Most useful for unit and functional testing.|
 |`default_options`|Allows you to set default values for the `mail` method options (`:from`, `:reply_to`, etc.).|
 
-For a complete writeup of possible configurations see the
-[Configuring Action Mailer](configuring.html#configuring-action-mailer) in
-our Configuring Rails Applications guide.
+Para uma descrição completa das configurações possíveis, consulte o
+[Configuring Action Mailer](configuring.html#configuring-action-mailer) dentro  
+nosso guia de configuração de aplicativos Rails.
 
-### Example Action Mailer Configuration
+### Exemplo de configuração do Action Mailer
```

Comment:
```suggestion
### Exemplo de configuração do _Action Mailer_
```

Acho que podemos seguir o padrão um pouco acima e deixar essa parte itálica o que acha?